terraform/Makefile

```diff
@@ -30,10 +30,16 @@ plan: validate
 
 .PHONY: apply
 apply: validate rename_resources
+	$(if $(findstring 1,$(AZUL_PRIVATE_API)), \
```

Comment:
The use of findstring is potentially dangerous. Is there no equality operator in GNU make? Look for `ifeq`